daq/host.py

```diff
@@ -699,6 +699,12 @@ def _get_switch_config(self):
             'password': self.switch_setup.get('password')
         }
 
+    def _get_usi_config(self):
+        return {
```

Comment:
how is this different than just returning self._usi_config?  The naming of things suggest one of two things 1) there's a comment required b/c the "why" is not at all clear, 2) this code is highly redundant and can be greatly simplified.

Reply:
I wanted to make sure the keys were always there since usi_config can be empty but it looks like jq doesn't fail if it's indexing a key that doesn't exist.